src/runtime/basic-runtime.js

```diff
@@ -266,6 +266,13 @@ export class BasicRuntime {
       runtime.processEntitlements()
     );
   }
+
+  /** Sets the callback when the offers flow is requested. */
+  setOnOffersFlowRequest(callback) {
```

Comment:
Is there a need to expose this? It seems that calling this would break the intended behavior implemented in the ConfiguredBasicRuntime constructor.

Reply:
Removed.